### PR TITLE
fix(UI): Handle ack and dt default options

### PR DESCRIPTION
**Pull request by Jordan91**
## Description

Handle ack and dt default options

## Type of change

- [x] Patch fixing an issue (non-breaking change)

### CODE REVIEW

File: src/Centreon/Application/Controller/Administration/ParametersController.php [hunk 1]
```diff
@@ -74,11 +87,25 @@ public function getParameters(): View
         $downtimeDuration = '';
         $downtimeScale = '';
         $refreshInterval = '';
+        $isAcknowledgementPersistent = false;
```
Comment (adr-mo) [inline src/Centreon/Application/Controller/Administration/ParametersController.php:90]:
```suggestion
        $isAcknowledgementPersistent = true;
```

File: src/Centreon/Application/Controller/Administration/ParametersController.php [hunk 2]
```diff
@@ -74,11 +87,25 @@ public function getParameters(): View
...
+        $isAcknowledgementSticky = false;
+        $isAcknowledgementNotify = true;
```
Comment (adr-mo) [inline src/Centreon/Application/Controller/Administration/ParametersController.php:92]:
```suggestion
        $isAcknowledgementNotify = false;
```

File: src/Centreon/Application/Controller/Administration/ParametersController.php [hunk 3]
```diff
@@ -74,11 +87,25 @@ public function getParameters(): View
         $downtimeDuration = '';
         $downtimeScale = '';
         $refreshInterval = '';
+        $isAcknowledgementPersistent = false;
+        $isAcknowledgementSticky = false;
```
Comment (adr-mo) [inline src/Centreon/Application/Controller/Administration/ParametersController.php:91]:
```suggestion
        $isAcknowledgementSticky = true;
```

File: src/Centreon/Application/Controller/Administration/ParametersController.php [hunk 4]
```diff
@@ -74,11 +87,25 @@ public function getParameters(): View
...
+        $isAcknowledgementNotify = true;
+        $isAcknowledgementWithServices = true;
+        $isAcknowledgementForceActiveChecks = false;
+        $isDowntimeFixed = true;
+        $isDowntimeWithServices = false;
```
Comment (adr-mo) [inline src/Centreon/Application/Controller/Administration/ParametersController.php:96]:
```suggestion
        $isDowntimeWithServices = true;
```

File: src/Centreon/Application/Controller/Administration/ParametersController.php [hunk 5]
```diff
@@ -92,6 +119,27 @@ public function getParameters(): View
                 case self::DEFAULT_REFRESH_INTERVAL:
                     $refreshInterval = $option->getValue();
                     break;
+                case self::DEFAULT_ACKNOWLEDGEMENT_PERSISTENT:
+                    $isAcknowledgementPersistent = (int) $option->getValue() === 1 ? true : false;
```
Comment (adr-mo) [inline src/Centreon/Application/Controller/Administration/ParametersController.php:123]:
```suggestion
                    $isAcknowledgementPersistent = (int) $option->getValue() === 1;
```

Comment (adr-mo) [reply src/Centreon/Application/Controller/Administration/ParametersController.php:123]: Do the same for the others

**Review by jeremyjaouen — APPROVED** [review]

**Review by kduret — APPROVED** [review]

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 6]
```diff
@@ -16189,3 +16189,6 @@ msgstr "Votre clé d'autologin doit être différente de votre mot de passe actu
 
 msgid "Your new password and autologin key must be different"
 msgstr "Votre nouveau mot de passe et votre clé d'autologin doivent être différents"
+
+# msgid "Force active checks"
+# msgstr "Forcer les contrôles actif"
```
Comment (kduret) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:16194]:
```suggestion
# msgstr "Forcer les contrôles actifs"
```

File: src/Centreon/Application/Controller/Administration/ParametersController.php (hunk 5 above)
Comment (Jordan91) [reply src/Centreon/Application/Controller/Administration/ParametersController.php]: Thank's

**Pull request merged**